### ISSUE TITLE
SA1516 is reported between extern alias and global using

### ISSUE THREAD

**Issue by Arthri** [DotNetAnalyzers/StyleCopAnalyzers#3853]
The following snippet doesn't raise SA1516 "Elements should be separated by blank line"
```cs
extern alias V;
using System.ComponentModel;
```
However, the next snippet does
```cs
extern alias V;
global using System.Buffers; // SA1516
using System.ComponentModel;
```

I believe that the behavior of SA1516 should remain consistent between the two snippets

**Comment by bjornhellander** [comment]
Unable to reproduce using the latest beta

**Comment by bjornhellander** [comment]
The issue with SA1210 is closely related to #3404. It happens when mixing global usings with normal usings in the same file. As stated in that issue: due to the complexity of a number of related analyzers, this might not be fixed anytime soon. The recommendation is to keep all your global usings in one place, to make the them easier to maintain. Global usings can be placed in the project file if you want. Another alternative is to keep them in a separate source file, called GlobalUsings.cs or something like that.

**Comment by bjornhellander** [comment]
I do however get SA1516 in all your examples, so I still don't see an inconsistency there. SA1516 simply wants a blank line between externs and usings.
